.pre-commit-config.yaml

```diff
@@ -30,3 +31,15 @@ repos:
     hooks:
       - id: pyupgrade
         args: ["--py3-plus"]
+# https://pre-commit.ci/
+ci:
+    autofix_commit_msg: |
+        [pre-commit.ci] auto fixes from pre-commit.com hooks
+
+        for more information, see https://pre-commit.ci
+    autofix_prs: false
+    autoupdate_branch: 'develop'
+    autoupdate_commit_msg: '[pre-commit.ci] pre-commit autoupdate'
+    autoupdate_schedule: monthly
+    skip: [flake8]
```

Comment:
Just wanted to touch on a couple things here.
Want to initially role this out without the autofix_prs.

I'm setting it to skip flake8 for 2 reasons...

We already have a linting CI step in azure.
Unlike most pre-commit hooks, it doesn't actually fix the issue. (although neither do check-yaml/check-json)

Reply:
Could we leave a comment in the YAML noting why we skip? Makes sense to me but it might be good to have this context in the config.